pillar/beacons.sls

```diff
@@ -2,6 +2,7 @@ beacons:
   network_settings:
     interval: 5
     # monitorize any change in IP addresses in eth0
+    # TODO: make this interface configurable
```

Comment:
Can you file a bug for that, so we don't forget about it?

Reply:
Filed [here](https://bugzilla.suse.com/show_bug.cgi?id=1063709)